reconcile/sql_query.py

```diff
@@ -0,0 +1,228 @@
+import time
+
+import jinja2
+import ruamel.yaml as yaml
+
+from reconcile import openshift_base
+from reconcile import queries
+
+from utils.oc import OC_Map
+from utils.state import State
+from utils.openshift_resource import OpenshiftResource
+
+
+QONTRACT_INTEGRATION = 'sql-query'
+JOB_TTL = 604800  # 7 days
+JOB_TEMPLATE = """
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ JOB_NAME }}
+spec:
+  template:
+    metadata:
+      name: {{ JOB_NAME }}
+    spec:
+      containers:
+      - name: {{ JOB_NAME }}
+        image: quay.io/app-sre/{{ ENGINE }}:{{ENGINE_VERSION}}
+        command:
+          - "psql"
+        args:
+          - "{{ ENGINE }}://$(db.user):$(db.password)@\
+$(db.host):$(db.port)/$(db.name)"
+          - "--command"
+          - "{{ QUERY }}"
+        env:
+          {% for key, value in DB_CONN.items() %}
+          {% if value is none %}
+          # When value is not provided, we get it from the secret
+          - name: {{ key }}
+            valueFrom:
+              secretKeyRef:
+                name: {{ SECRET_NAME }}
+                key: {{ key }}
+          {% else %}
+          # When value is provided, we get just use it
+          - name: {{ key }}
+            value: {{ value }}
+          {% endif %}
+          {% endfor %}
+      restartPolicy: Never
+"""
+
+
+def get_tf_resource_info(namespace, identifier):
+    """
+    Extracting the terraformResources information from the namespace
+    for a given identifier
+
+    :param namespace: the namespace dictionary
+    :param identifier: the identifier we are looking for
+    :return: the terraform resource information dictionary
+    """
+    tf_resources = namespace['terraformResources']
+    for tf_resource in tf_resources:
+        if tf_resource['identifier'] != identifier:
+            continue
+        if tf_resource['provider'] != 'rds':
+            continue
+
+        defaults = queries.get_resource(tf_resource['defaults'])
+        return {
+            'cluster': namespace['cluster']['name'],
+            'output_resource_name': tf_resource['output_resource_name'],
```

Comment:
`output_resource_name` is not always defined.
we should either copy the logic here, it's fairly simple:
https://github.com/app-sre/qontract-reconcile/blob/master/utils/terrascript_client.py#L1059-L1062

Reply:
oh, ok, thanks! fixed.